python_modules/libraries/dagster-databricks/dagster_databricks/ops.py

```diff
@@ -135,3 +136,239 @@ def create_ui_url(databricks_client, op_config):
         return "https://{host}/?o={workspace_id}#joblist".format(
             host=host, workspace_id=workspace_id
         )
+
+
+def create_databricks_run_now_op(databricks_job_id: int, **kwargs) -> OpDefinition:
```

Comment:
I wonder if this should take a `databricks_job_configuration` like the other function? Seems nice to not need to use `configured` to specify `python_params`?